### ISSUE TITLE
How to detect certain actions in my own video data

### ISSUE THREAD

**Issue by west-i** [wei-tim/YOWO#17]
What steps do I need to take if I want to detect a certain action in my video data.

**Comment by wei-tim** [comment]
@west-i 
Hi, thanks for your interest.
First you have to check whether the action that you are going to detect is in the datasets (JHMDB-21 or UCF101-24). 
If so, you can directly use our pretrained model to detect this specific action. 
Otherwise you can use our method to train your own model on your specific dataset.

**Comment by west-i** [comment]
@wei-tim 
Thanks you for your answer.

The action that i want to detect is not in the datasets(JHMDB-21 or UCF101-24).
I have the following questions：

1.How should I annotate the new action in the video to train my detection model. 
2.What needs to be modified next to match my data? 

Beginners in this area, hope to get your answers, thank you very much!

**Comment by abhigoku10** [comment]
@west-i  can you please share your pre-trained weights

**Comment by okankop** [comment]
@west-i 
1. you should annotate your data similar to UCF101-24 annotations.
2. You just need to change the number of classes in the network architecture and you are ready to train.

**Comment by okankop** [comment]
@abhigoku10 you can simply follow the test procedure applied in the training. Please investigate the "def test(epoch)" function in file https://github.com/wei-tim/YOWO/blob/master/train.py

**Comment by west-i** [comment]
@okankop 
Thanks you for your answer.

Is there a labeling tool I can use to label my videos?

**Comment by okankop** [comment]
@west-i 
Labelme (https://github.com/wkentaro/labelme) is a very nice labelling tool. You can use that.

**Comment by okankop** [comment]
I believe issue is resolved!
